WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -764,6 +777,16 @@ - (UITableViewCell *)configureGeolocationCellForIndexPath:(NSIndexPath *)indexPa
     return cell;
 }
 
+- (UITableViewCell *)configureExcerptCellForIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [self getWPTableViewCell];
```

Comment:
I know this wasn't introduced in your PR, but... (nipticky request!) can we rename `getWPTableViewCell` into something like... `newWPTableViewCell` perhaps?

(Old school memory management conventions!)